File: Freelance/Medie-Superiori/Informatica/Gabriele R/05-03/Es1.py
#Dichiarazione delle variabili
temperatura = 0
somma = 0
massima = 0
minima = 0
giorni = 0
giorni_sopra = 0
giorni_sotto = 0
#Inserimento del mese
mese = input("Inserisci il mese: ")
#Inserimento del numero di giorni
giorni = int(input("Inserisci il numero di giorni: "))
#Inserimento delle temperature
for i in range(1, giorni + 1):
    temperatura = int(input(f"Inserisci la temperatura del giorno {i}: "))
    somma += temperatura
    if temperatura > massima:
        massima = temperatura
    if temperatura < minima:
        minima = temperatura
    if temperatura > 30:
        giorni_sopra += 1
    if temperatura < 10:
        giorni_sotto += 1
#Calcolo della temperatura media
media = somma / giorni
#Stampa dei risultati
print(f"La temperatura media di {mese} è {media}")
print(f"La temperatura massima registrata è {massima}")
print(f"La temperatura minima registrata è {minima}")
print(f"I giorni sopra i 30 gradi sono {giorni_sopra}")
print(f"I giorni sotto i 10 gradi sono {giorni_sotto}")

File: Ripetizioni private/Medie-Superiori/Informatica/Ettore/24-04/Es1.java
import java.util.Scanner;
public class Es1 {
    public static void main(String[] args) {
        int numero, somma;
        Scanner input = new Scanner(System.in);
        System.out.println("Inserisci il numero: ");
        numero = input.nextInt();

        if(numero % 2 == 0){
            int i = 2;
            while(i <= n){
                somma += i;
                i += 2;
            }
        }
        if(numero % 2 == 1){
            int i = 1;
            while(i <= n){
                somma += i;
                i += 2;
            }
        }
    System.out.println("La somma è: " + somma);
    }
}
